perfkitbenchmarker/linux_benchmarks/object_storage_service_benchmark.py

```diff
@@ -793,27 +795,61 @@ def LoadWorkerOutput(output):
   return start_times, latencies, sizes
 
 
-def MultiStreamRWBenchmark(results, metadata, vms, command_builder,
-                           service, bucket_name, regional_bucket_name):
+def _RunMultiStreamProcesses(vms, command_builder, cmd_args,
+                             streams_per_vm, num_streams):
+
+  """Runs all of the multistream read or write processes and doesn't return
+     until they complete.
+
+  Args:
+    vms: the VMs to run the benchmark on.
+    command_builder: an APIScriptCommandBuilder.
+    cmd_args: arguments for the command_builder.
+    streams_per_vm: number of threads per vm.
+    num_streams: total number of threads to launch.
+  """
+
+  output = [None] * num_streams
+
+  def _StartMultiStreamProcess(proc_idx):
```

Comment:
Suggestion: rename this to `RunOneProcess`. (I know I chose the old name, my fault.)